credentials/tls_ext_test.go

```diff
@@ -236,3 +238,72 @@ func (s) TestTLS_CipherSuitesOverridable(t *testing.T) {
 		t.Fatalf("EmptyCall err = %v; want <nil>", err)
 	}
 }
+
+// TestTLS_DisabledALPN tests the behaviour of a gRPC client when connecting to
+// a server that doesn't support ALPN.
```

Comment:
This comment needs updating since we are not creating a grpc client anymore and only testing the ClientHandshake method on the TLS creds.

Reply:
Missed that, fixed now.